### PR TITLE
Promote staging to prod

### PR DESCRIPTION
**Pull request by cryptoseneca**
This PR pushes the updated navigation bar feature to production:
- #322 
- #335

### CODE REVIEW

File: packages/nouns-webapp/src/components/NavBarTreasury/index.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+interface NavBarTreasuryProps {
+  treasuryBalance: string;
+  treasuryStyle: NavBarButtonStyle;
+}
+
+const NavBarTreasury: React.FC<NavBarTreasuryProps> = props => {
+  const { treasuryBalance, treasuryStyle } = props;
+
+  let treasuryStyleClass;
+  switch (treasuryStyle) {
```
Comment (gitblamecarrot) [inline packages/nouns-webapp/src/components/NavBarTreasury/index.tsx:13]: Smell here: effectively duplicate code translating a `NavBarButtonStyle` to classes with `packages/nouns-webapp/src/components/NavBarButton/index.tsx`. Should be addressable with a single helper and a common module.

Doesn't need to block promotion but should be addressed later.

File: packages/nouns-webapp/src/components/NavWallet/index.tsx [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+        aria-labelledby={props.labeledBy}
+      >
+        <div>
+          <div
+            onClick={switchWalletHandler}
+            className={clsx(
+              classes.dropDownTop,
+              classes.button,
+              classes.switchWalletText,
+              usePickByState(
```
Comment (gitblamecarrot) [inline packages/nouns-webapp/src/components/NavWallet/index.tsx:145]: "Helper functions that contain hooks as parameters to functions in props" is a mouthful, which is a good sign it's probably a smell as well.

Also not blocking but could use some attention.

File: packages/nouns-webapp/src/components/WalletConnectModal/index.tsx [hunk 3]
```diff
@@ -31,7 +30,7 @@ const WalletConnectModal: React.FC<{ onDismiss: () => void }> = props => {
       <WalletButton
         onClick={() => {
           const fortmatic = new FortmaticConnector({
-            apiKey: 'pk_live_60FAF077265B4CBA',
+            apiKey: 'pk_test_FB5E5C15F2EC5AE6',
```
Comment (gitblamecarrot) [inline packages/nouns-webapp/src/components/WalletConnectModal/index.tsx:33]: Regression to a test Fortmatic key, this one is blocking.

**Review by gitblamecarrot — CHANGES_REQUESTED** [review]
There's a regression in the Fortmatic key that should block promotion to prod.

Other notes are just smells.

**Comment by cryptoseneca** [comment]
Closed due to #308 including updates to nav bar. dev's review comments are now in #343

**Pull request closed without merging**